.settings/com.google.gdt.eclipse.core.prefs

```diff
@@ -1,5 +1,7 @@
-eclipse.preferences.version=1
+#Tue Dec 22 10:52:24 CST 2015
 jarsExcludedFromWebInfLib=
+warSrcDir=src/main/webapp/
+lastWarOutDir=/Users/mpayne/Study2/Java/todomvc-tessell/war
```

Comment:
Hm, that is unfortunate if it really wants to put this lastWarOutDir in a settings file.